jupyterhub/handlers/login.py

```diff
@@ -105,6 +105,8 @@ def _render(self, login_error=None, username=None):
                     'next': self.get_argument('next', ''),
                 },
             ),
+            "authenticator_request_otp": self.settings['request_otp'],
+            "authenticator_otp_prompt": self.settings['otp_prompt'],
```

Comment:
These are attributes on the Authenticator:

```suggestion
            "authenticator_request_otp": self.authenticator.request_otp,
            "authenticator_otp_prompt": self.authenticator.otp_prompt,
```


By exposing a `config=True` trait on a class, configuration ultimately sets the attribute values on the instances themselves. It's like giving users the ability to do `self.authenticator.request_otp = "x"` on an approved set of attributes in their config files.